### ISSUE TITLE
NPM doesn't work out of the box

### ISSUE THREAD

**Issue by rurabe** [react-bootstrap/react-bootstrap#538]
I see other people have had this or similar errors, but I cant get this working (even if I link directly to the repo).

`npm install --save react-bootstrap`

then in a component

`var Bootstrap = require('react-bootstrap');`

then gulp

```
gulp
...
events.js:72
        throw er; // Unhandled 'error' event
              ^
```

Is there something I'm missing? Am I doing it wrong?

**Comment by dozoisch** [comment]
Do you have the stack trace? That error could be related to pretty much anything. Also, what npm, node, etc. versions are you using? Can you make a small example we can reproduce?

**Comment by rurabe** [comment]
So sorry, this seemed to be an OSX configuration issue that I just happened to hit as I required `bootstrap-select`. Figured it out when I went to make a small sample app for you and it worked fine. 

Just for posterity, the issue was the number of files OSX can open in parallel, and Browserify running up against that limit, as noted in [this issue](https://github.com/substack/node-browserify/issues/431). Also, there was more stack trace but I ignored it, stupidly:

```
events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: EMFILE, open '/Users/<PATH>/node_modules/react/package.json'
```

Changed the limit on files in my bash_profile and everything works now. Sorry!
